src/interfaces/IArrangerConduit.sol

```diff
@@ -234,10 +251,11 @@ interface IArrangerConduit is IAllocatorConduit {
     /**
      * @notice Draw funds from the contract to the Arranger.
      * @dev    Only the Arranger is authorized to call this function.
```

Comment:
Would also specify that the `destination` needs to be whitelisted as a `broker`.

Reply:
Updated